tools/ew.py

```diff
@@ -93,6 +93,22 @@ def platform_to_qemu_options(platform, machine, processor):
 		return 'system-x86_64', pc_options(64)
 	elif platform == 'arm32':
 		return 'system-arm', '-M integratorcp'
+	elif platform == 'arm64':
+		# Check that ROM image is present. Provide the user with
+		# appropriate steps to fix this problem.
```

Comment:
It would be nice to support placing the image in some location where QEMU stores other firmware images, so that it does not have to be in the HelenOS root.